ices/64450.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+tmp="$(mktemp -d)"
+
+if [[ ! $tmp || ! -d $tmp ]]
+then
+    echo "Could not create temporary directory"
+    exit 1
+fi
+
+cleanup() {
+    rm -r "$tmp"
+}
+
+trap cleanup EXIT
+
+cd "$tmp"
+
+git clone https://github.com/thiolliere/debug-inconsistence.git && cd debug-inconsistence/runtime
```

Comment:
Yeah I don't think ones should be added that are not self contained

Reply:
Mind if I _make_ them (64450, 66003, and 66922) self-contained?